_pages/rentals/intervale-manual.md

```diff
@@ -37,6 +37,8 @@ Intervale is located a few miles north of North Conway, NH. The exact location o
 
 Intervale is not road-accessible. Therefore, we have an easement which gives us a Right of Way (RoW) to access our property. Our RoW passes by a pair of other homes and directly adjacent the property of a third. It is important that you follow all of the rules below regarding our RoW when you access our property. We must respect our neighbors at all times. Failure to follow these procedures constitutes trespassing.
 
+Please note that accessing the cabin involves a moderate 250 meter hike, with uneven terrain and one rather steep hill. In winter, it can be useful to have spikes or some additional traction to assist with footing.
```

Comment:
250m feels like a low estimate, even without counting the walk around the road. Maybe we should mention that it's usually a 10 to 15 minutes hike from the parking lot?